ui/src/app/services/interceptors/csrf.interceptor.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
```

Comment:
Not considering `HttpClientXsrfModule` and cookie-to-header method because cookie can't be read from different port in live-serving mode. The current localStorage-to-header method is equally secure.